lib/query-options.js

```diff
@@ -1,6 +1,7 @@
 const Long = require("long");
 const _execOptions = require("./execution-options");
 const { longToBigint } = require("./new-utils");
+const rust = require("../index");
 
```

Comment:
:question: in commit message `Options wrapping (caching)`: `across the langues` ? Do you mean `across the languages`?